utilities/xmlrpcpp/include/xmlrpcpp/XmlRpcClient.h

```diff
@@ -130,6 +145,14 @@ namespace XmlRpc {
     // Event dispatcher
     XmlRpcDispatch _disp;
 
+  protected:
+    //! general timeout passed to the work method of the dispatcher, to allow avoiding calls blocking forever
+    static double executeTimeout; // s, -1 means infinite wait
```

Comment:
why this is declared as `protected`, not `private`?

Reply:
I aligned with the rest of the code, I didn't see any reasons to make it private instead of protected.
Maybe the code needs some fixing, since most members are public.